backend/engine/utils/plugin.py

```diff
@@ -240,7 +240,15 @@ def is_plugin_disabled(settings: dict) -> bool:
     return True
 
 
-def run_plugin(plugin, scan, scan_images, depth=None, include_dev=False, features=None, services=None) -> Result:
+def run_plugin(
+    plugin: str,
+    scan: Scan,
+    scan_images,
+    depth: Optional[str] = None,
+    include_dev=False,
```

Comment:
include_dev should just be True or False, could we explicitly call this out as expecting a `bool`?

Reply:
This is actually a weird case.  The type of `include_dev` can actually be `Union[str, bool]`, with an extra potential `Unknown` in there because AFAICT it's not validated or constrained to `bool` when passed-in.

Since this PR is intended to skip over anything that would require a change of code outside this one file, this param was left as-is for now.